### PR TITLE
Detect removed thermostats

### PR DESCRIPTION
**Pull request by bouwew**
Detect thermostat devices removed from a location in the Plugwise App.

### CODE REVIEW

**Comment by bouwew** [comment]
@CoMPaTech what is a better name for a group of switches? Group Switch, Switch Group or maybe Switch group? I think I don't like Group Switch...

**Comment by CoMPaTech** [comment]
> @CoMPaTech what is a better name for a group of switches? Group Switch, Switch Group or maybe Switch group? I think I don't like Group Switch...

switchgroup sounds more natural and to the point

File: plugwise/helper.py [hunk 1]
```diff
@@ -314,20 +314,21 @@ def _locations_legacy(self):
         Create locations for legacy devices.
         """
         appliances = set()
-        self._home_location = 0
+        fake_loc_id = "0000aaaa0000aaaa0000aaaa0000aa00"
```
Comment (CoMPaTech) [inline plugwise/helper.py:317]: maybe 'const' default this?

File: tests/test_smile.py [hunk 2]
```diff
@@ -566,13 +567,15 @@ async def test_connect_legacy_anna(self):
             },
             # Central
             "04e4cbfe7f4340f090f85ec3b9e6a950": {
+                "location": "0000aaaa0000aaaa0000aaaa0000aa00",
```
Comment (CoMPaTech) [inline tests/test_smile.py:570]: and retrieve this from `const` as well

**Review by CoMPaTech — APPROVED** [review]
See comment on `const`ing to avoid rewriting them

File: plugwise/helper.py (hunk 1 above)
Comment (bouwew) [reply plugwise/helper.py]: Implemented.

File: tests/test_smile.py (hunk 2 above)
Comment (bouwew) [reply tests/test_smile.py]: Implemented too.

**Pull request merged**